cardano-crypto-class/src/Cardano/Crypto/VRF/Class.hs

```diff
@@ -218,6 +227,17 @@ instance (VRFAlgorithm v, Typeable a) => ToCBOR (CertifiedVRF v a) where
       toCBOR (certifiedNatural cvrf) <>
       encodeCertVRF (certifiedProof cvrf)
 
+  encodedSizeExpr size proxy =
+        1
+      + encodedSizeExpr    size ((toWord64 . certifiedNatural) <$> proxy)
```

Comment:
This is using a proxy at type Word64 right? But it's actually a natural.

So it really is a `Natural` but the max size of that `Natural` is given in the class above by:
```
  maxVRF :: proxy v -> Natural
```

Reply:
I see,I misinterpreted it. I will fix this.